cmd/tailscaled/tailscaled.go

```diff
@@ -679,6 +679,10 @@ func tryEngine(logf logger.Logf, sys *tsd.System, name string) (onlyNetstack boo
 		DriveForLocal: driveimpl.NewFileSystemForLocal(logf),
 	}
 
+	if sys.InitialConfig != nil {
+		conf.StaticEndpoints = sys.InitialConfig.Parsed.StaticEndpoints
+	}
```

Comment:
Rather than plumb this down once, can we make `func (b *LocalBackend) ReloadConfig() (ok bool, err error)` notify the downstream things (like magicsock) of this? 

I'd also then pass down a view.Slice[netip.AddrPort] instead of a mutable slice, to make ownership clear.

Note that `ReloadConfig` isn't used today during the startup path, so it'd need to be split into `ReloadConfig` + `setConfigLocked` halves, and use `setConfigLocked` during the LocalBackend constructor.

Reply:
@bradfitz I do that when local backend reloads config as well, this was just the inital set https://github.com/tailscale/tailscale/pull/12882/commits/7fb3a014cfd08c031548c6637446e3ba756f0102#diff-314ba0d799f70c8998940903efb541e511f352b39a9eeeae8d475c921d66c2acR620-R632

Reply:
Ah you meant _only_ do it via ReloadConfig